implementation/eCatalogue/README.txt

```diff
@@ -0,0 +1,22 @@
+Description: The eCatalogue module contains classes related to the concept of an eCatalogue. It also 
```

Comment:
hi, there is an 'It also' that appears not to belong there.

Reply:
Got it, thanks.